### ISSUE TITLE
Cannot set table cell margins

### ISSUE THREAD

**Issue by kristofferjalen** [EvotecIT/OfficeIMO#116]
I understand table cell margins should be set like this:

```c#
var tableCellMarginDefault = new TableCellMarginDefault(
   new TopMargin { Width = "21", Type = TableWidthUnitValues.Pct },
   new StartMargin { Width = "21", Type = TableWidthUnitValues.Pct },
   new BottomMargin { Width = "21", Type = TableWidthUnitValues.Pct },
   new EndMargin { Width = "21", Type = TableWidthUnitValues.Pct });

var tableProperties = new TableProperties(tableCellMarginDefault);
```

I cannot find a way to assign the `TableProperties` instance to a table though.

**Comment by PrzemyslawKlys** [comment]
I don't think we expose margins directly yet. We only support choosing one of the built-in styles which already have the required margins set. This needs to be added/exposed.

**Comment by PrzemyslawKlys** [comment]
Additional questions for it:
- you're after setting totally custom style or modifying existing styles? 
- what would be expectation when you have table with style `GridTable6ColorfulAccent1` that you "modify" on the table - would you want it to apply to all tables using same style or the style should now be cloned and become new style? 

The more think about it, the more complicated it becomes to implement.

**Comment by PrzemyslawKlys** [comment]
This PR solves it (I think):
- https://github.com/EvotecIT/OfficeIMO/pull/129
